src/sentry/workflow_engine/models/alertrule_workflow.py

```diff
@@ -35,3 +36,15 @@ class Meta:
                 name="rule_or_alert_rule_workflow",
             ),
         ]
+        indexes = [
+            models.Index(
+                fields=["rule_id"],
+                name="idx_arw_rule_id",
+                condition=Q(rule_id__isnull=False),
+            ),
+            models.Index(
+                fields=["alert_rule_id"],
+                name="idx_arw_alert_rule_id",
+                condition=Q(alert_rule_id__isnull=False),
+            ),
+        ]
```

Comment:
It might be worth just having this condition generally, we'll probably want to join on these cols other times too. Up to you though.

Reply:
eh, i'll just remove the conditions. i was just trying to save a couple cycles when creating the indices, but i think you're right it's better to have it be more general use case.